.gitignore

```diff
@@ -11,4 +11,5 @@
 /public/assets
 /tags
 /tmp/*
-/node_modules
\ No newline at end of file
+/node_modules
+/sendgrid.env
```

Comment:
That was for my local environment - ended up going a different route anyway, so I'll remove it.